xml/admin_ceph_upgrade.xml

```diff
@@ -485,6 +485,24 @@ mds standby for name = mds.456-another-mds
     Check that required repositories are configured on each cluster's node. To
```

Comment:
> each cluster's node

...(?)

What does this use of node in the singular mean? 1 node in each? All nodes in each cluster seems more likely to me, but I am not sure. If so, how about "each cluster's nodes"?

Reply:
fixed